saleor/graphql/warehouse/tests/test_warehouse.py

```diff
@@ -374,6 +374,42 @@ def test_mutation_create_warehouse(
     assert created_warehouse["slug"] == warehouse.slug
 
 
+def test_mutation_create_warehouse_does_not_create_when_name_is_empty_string(
+    staff_api_client, permission_manage_products, shipping_zone
+):
+    Warehouse.objects.all().delete()
```

Comment:
Why are you removing all the warehouses?

Reply:
Honestly, I copied the test above and changed just what I needed. Missed that line.